src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+
+  if (days >= 3) {
+    discount = 20;
```

Comment:
Create a variable for these 'magic numbers' and use it here

Reply:
you mean for a discount? or a days for discount